app/containers/message/Audio.js

```diff
@@ -197,7 +197,7 @@ class Audio extends React.Component {
 						onValueChange={this.onValueChange}
 						thumbImage={isIOS && { uri: 'audio_thumb', scale: Dimensions.get('window').scale }}
 					/>
-					<Text style={[styles.duration, { color: themes[theme].auxiliaryText }]}>{this.duration}</Text>
+					{paused ? <Text style={[styles.duration, { color: themes[theme].auxiliaryText }]}>{this.duration}</Text> : <Text style={[styles.duration, { color: themes[theme].auxiliaryText }]}>{formatTime(currentTime)}</Text>}
```

Comment:
The whole `<Text style={[styles.duration, { color: themes[theme].auxiliaryText }]}>...</Text>` is repeated twice.
Why not creating a new variable and use it like this?
```
<Text style={[styles.duration, { color: themes[theme].auxiliaryText }]}>
    {newVariable}
</Text>
```